src/session1_homework/Exercise1.java

```diff
@@ -0,0 +1,18 @@
+package session1_homework;
+
+public class Exercise1 {
+    // 1. Simple Output Program: Write a program that prints you name, age, and favorite color on three separate lines.
+
```

Comment:
Please try to avoid too many empty rows left in the code.
If you want to annotate a method with the description of the problem, you can use java doc /** ... */ or multiple comment lines /* ... */. It will give a better look, it will be easier to read and we'll follow code formatting standards.